ui/src/tempVars/components/TemplateVariableEditor.tsx

```diff
@@ -291,7 +321,16 @@ class TemplateVariableEditor extends PureComponent<Props, State> {
       return
     }
     const {onUpdate, onCreate, notify} = this.props
-    const {nextTemplate, isNew} = this.state
+    const {
+      nextTemplate,
+      isNew,
+      isDynamicSourceSelected,
+      selectedSource,
+    } = this.state
+    nextTemplate.sourceID = '0' // setting to 0 b/c an empty string returns an unparseable json error from the BE
```

Comment:
should this be an int?

https://github.com/influxdata/chronograf/pull/5346/files#diff-5924b93d694832218a14e67d800c1016R188

Reply:
Long story short, no, since it’s converted into a string from the api when it’s returned